packages/dai-plugin-migrations/src/migrations/ScdToMcdCdp.js

```diff
@@ -0,0 +1,14 @@
+export default class ScdToMcdCdp {
```

Comment:
what would you think about calling this `SingleToMulti` instead? rolls off the tongue better

Reply:
Yea, agreed.

Reply:
Saving the rename for a later PR?

Reply:
More like I thought I merged it but apparently didn't? 🤔